serbian/burekSir/pocetak/pocetak.md

```diff
@@ -0,0 +1,20 @@
+Čemu [bes](bes.md), čemu [ozlojađenost](nezadovoljstvo.md). Pekara može i sme da ostane bez mog omiljenog bureka. Ljudi oko mene mogu da se smeju. Loše stvari mogu da se dese. Dobre stvari mogu da se dese. 

```

Comment:
This line contains more than one link.  Please put links on separate lines with a blank line in between so that options will be visually separated.